src/python/WMComponent/AgentStatusWatcher/AgentStatusPoller.py

```diff
@@ -106,6 +107,12 @@ def algorithm(self, parameters):
         get information from wmbs, workqueue and local couch
         """
         try:
+            config_path = os.path.join(os.environ.get('config', ''), 'config.py')
```

Comment:
I've changed my mind 5 times this morning :) I leave it up to you, surprise me LoL